app/views/application/_page_actions.html.haml

```diff
@@ -6,6 +6,7 @@
   - if show_new_page_link?
     %li
       = link_to 'New page', prepare_editorial_section_nodes_path(@section, parent_id: @node)
+      = link_to 'New news article', editorial_new_news_article_path(publisher: @section.id)
```

Comment:
The style isn't affected but you're right in that it's probably more semantically correct for it to have its own link.